distribution/docs/pom.xml

```diff
@@ -18,8 +18,8 @@
 
     <parent>
         <groupId>ddf</groupId>
-        <artifactId>ddf-parent</artifactId>
-        <version>3.0.2-SNAPSHOT</version>
+        <artifactId>distribution</artifactId>
```

Comment:
It shouldn't be an issue because `distribution/pom.xml` inherits from ddf-parent. So `distribution/docs/pom.xml` should indirectly inherit from ddf-parent. Also I ran `mvn clean install | grep checkstyle` and got:

```
[INFO] --- maven-checkstyle-plugin:2.15:check (checkstyle-check) @ docs ---
[INFO] --- maven-checkstyle-plugin:2.15:check (checkstyle-check-xml) @ docs ---
```

So it is using the correct version of checkstyles.